lib/posix-event/epoll.c

```diff
@@ -371,3 +373,17 @@ int epoll_pwait(int epfd, struct epoll_event *events, int maxevents,
 	return ret;
 }
 #endif /* UK_LIBC_SYSCALLS */
+
+static int epoll_mount_init(void)
+{
+	epoll_mount.m_path = strdup("");
+	if (!epoll_mount.m_path)
+		return -ENOMEM;
+	epoll_mount.m_special = strdup("");
+	if (!epoll_mount.m_special) {
+		free(epoll_mount.m_path);
```

Comment:
I'd actually prefer a goto logic as we do in general for error exits. The reason is that this can be extended more easily later:

```c
static int epoll_mount_init(void)
{
	epoll_mount.m_path = strdup("");
	if (!epoll_mount.m_path)
		goto err_out;
	epoll_mount.m_special = strdup("");
	if (!epoll_mount.m_special) {
		git err_free_m_path;
	return 0;

err_free_m_path:
	free(epoll_mount.m_path);
err_out:
	return -ENOMEM;
}
```